docs/src/main/sphinx/connector/bigquery.rst

```diff
@@ -207,10 +207,10 @@ Retrieve all records stored in the partition ``_PARTITIONDATE = '2022-04-07'``::
 SQL support
 -----------
 
-The connector provides read and write access to data and metadata in
```

Comment:
"create" and "drop" table qualify as write access. How is that inaccurate?

Reply:
I can clarify further - the intent behind this change is that you cannot actually write new data of any kind, which gets a little lost when you say "it provides write access to data."